recipes/pymc3-ext/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "pymc3-ext" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pymc3_ext-{{ version }}.tar.gz
+  sha256: 0e1be0d1c7e05f639d0cad5e6f064041b4952404e2dad29c4d65ef8e66c5409f
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
```

Comment:
This run section is at odds with what is in the `install_requires` in the `setup.py`:

```
INSTALL_REQUIRES = [
    "numpy",
    "scipy",
    "theano>=1.0.4",
    "pymc3>=3.8",
    "fastprogress",
]
```

it may be that `pymc3` is pulling in all of the reqs (I didn't check), but generally we want anything in the `install_requires` listed here.

Reply:
Good point! I'll fix that, although probably by updating the pip specs! Thanks.